test/plugin/test_in_tail.rb

```diff
@@ -1878,13 +1878,6 @@ def test_z_refresh_watchers
     plugin.instance_eval do
       @pf = Fluent::Plugin::TailInput::PositionFile.load(sio, EX_FOLLOW_INODES, {}, logger: $log)
       @loop = Coolio::Loop.new
-      opened_file_metrics = Fluent::Plugin::LocalMetrics.new
-      opened_file_metrics.configure(config_element('metrics', '', {}))
-      closed_file_metrics = Fluent::Plugin::LocalMetrics.new
-      closed_file_metrics.configure(config_element('metrics', '', {}))
-      rotated_file_metrics = Fluent::Plugin::LocalMetrics.new
-      rotated_file_metrics.configure(config_element('metrics', '', {}))
-      @metrics = Fluent::Plugin::TailInput::MetricsInfo.new(opened_file_metrics, closed_file_metrics, rotated_file_metrics)
```

Comment:
Since this pull request adds `throttling_metrics`, it should be added here too.
On the other hand, I've noticed this initialization isn't needed originally since `create_driver` runs `configure` and it initializes `@metrics`. I overlooked it in reviewing #3504.

Reply:
I see
I didnt notice this when i was navigating through code